pkg/internal/goexec/offsets.json

```diff
@@ -44,7 +44,7 @@
       "conn": {
         "versions": {
           "oldest": "1.40.0",
-          "newest": "1.45.1"
```

Comment:
I'm not sure if we should modify this file manually. It's being updated thru this https://github.com/grafana/go-offsets-tracker

Reply:
I updated that file by running the command:
```bash
make update-offsets
```
Is that wrong? btw my current version is go 1.24.2